Jellyfin.Api/Helpers/StreamingHelpers.cs

```diff
@@ -210,7 +210,7 @@ public static async Task<StreamState> GetStreamingState(
                     && state.VideoRequest.VideoBitRate.Value >= state.VideoStream.BitRate.Value)
                 {
                     // Don't downscale the resolution if the width/height/MaxWidth/MaxHeight is not requested,
-                    // and the requested video bitrate is higher than source video bitrate.
+                    // and the requested video bitrate is greater than source video bitrate.
```

Comment:
This is a comparison between two actual quantities, so `greater than` is appropriate.